### ISSUE TITLE
Starting engine always takes two tries

### ISSUE THREAD

**Issue by pmlt** [turbulent/substance#5]
Reported by @epenner :

```
epenner@MacBook-Pro-97 ~/substance/work (master)  > substance engine launch work
Provisioning engine "work" with driver "virtualbox"
Checking for port conflicts
Determined SSH port as 4500
Configure machine shared folders
Configure machine profile
[work] ==> Updating network info from driver
[work] ==> Waiting for machine to boot...
Max retries exceeded (LinkRetriesExceeded)
epenner@MacBook-Pro-97 ~/substance/work (master)  > substance engine launch work
Provisioning engine "work" with driver "virtualbox"
[work] ==> Updating network info from driver
[work] ==> Registering engine as 'work.dev' in local hosts file
Hosts register work.dev: 172.21.21.2
'work.dev' found in local hosts file.
Engine "work" has been launched.
[work] ==> Starting all containers...
'dbmaster' has been started.
'cache' has been started.
'sessions' has been started.
'redis' has been started.
'qmgr' has been started.
'web' has been started.
'cron' has been started.
'logrotate' has been started.
'island' has been started.
```

**Comment by psaunders** [comment]
I spun up a new virtual machine for a project and can't actually get past the:

```
Max retries exceeded (LinkRetriesExceeded)
```

**Comment by bbeausej** [comment]
This was solved in the latest boxes (0.7 and up)
